troubleshooting/known_issues/Gigabye_Missing_Redfish_Data.md

```diff
@@ -0,0 +1,24 @@
+### Gigabyte BMC Missing Redfish Data
+
+Follow this procedure if you notice data from Gigabyte nodes are missing from Hardware State Manager(HSM) or other CSM tools.

```

Comment:
- Add space between "Manager" and "(HSM)"
- Change "are" to "is"

Reply:
Updated